Project.toml

```diff
@@ -16,13 +16,13 @@ Symbolics = "0c5d862f-8b57-4792-8d23-62f2024744c7"
 
 [compat]
 ADNLPModels = "0.4.0 - 0.8"
-ForwardDiff = "0.10"
+ForwardDiff = "0.10 - 1"
 JuMP = "1"
 MathOptInterface = "1"
-NLPModelsJuMP = "0.13"
-ReverseDiff = "1.14"
-Symbolics = "4.10.4 - 5.1"
-julia = "^1.6.0"
+NLPModelsJuMP = "^0"
+ReverseDiff = "1"
+Symbolics = "4 - 6"
+julia = "1"
```

Comment:
It's probably worth being more specific here like `1.10` or `lts`

Reply:
You actually want to maintain the LTS which is Julia 1.10 and if anything below doesn't work that's fine (we no longer test it).
You can simply write `julia = 1.10` and the CI should still be able to run

Reply:
Obviously next release is a breaking release.